src/libraries/System.Text.Json/tests/Common/RequiredKeywordTests.cs

```diff
@@ -664,6 +664,35 @@ public class ClassWithCustomRequiredPropertyName
             public required int PropertyWithInitOnlySetter { get; init; }
         }
 
+        [Fact]
+        public async Task DerivedClassWithRequiredProperty()
+        {
+            var value = new DerivedClassWithRequiredInitOnlyProperty { MyInt = 42, MyBool = true, MyLong = 4242 };
+            string json = await Serializer.SerializeWrapper(value);
+            Assert.Equal("""{"MyInt":42,"MyBool":true,"MyString":null,"MyLong":4242}""", json);
```

Comment:
Would it be possible to set a value to the `MyString` property to make sure it's being populated properly?

Reply:
Indeed.